test/serializer/optimized-functions/ObjectAssign6.js

```diff
@@ -0,0 +1,12 @@
+// does not contain:return 1
```

Comment:
I thought `does not contain` was added for these reasons? I'm happy to change it to something else. We'd need an assertion for if something is abstract, I'm guessing from your above comment?

Reply:
I think @sebmarkbage means that if you tried to assert the optimization doesn't break correctness, just executing the test and comparing `inspect` results (which happens anyway) would be sufficient. So this comment doesn't seem to add anything useful to the check. On the other hand, there are plausible optimizations (such as returning `1` early under a condition) that would break this test but would otherwise be totally valid. So again, seems like this line just adds friction.

Reply:
Okay, let me take another look at this.

Reply:
I've updated the condition to be:

`// does contain:.x;`